### DISCUSSION TITLE
Hex to image and image to hex

### DISCUSSION THREAD

**Discussion by cmallory183** [CorentinTh/it-tools#724] [Ideas]
I often find myself needing to convert an image to hex to store in a database. While this is simple enough in code, directly editing a database for one-off changes can be a hassle. Having an easy way to take an image and convert it to hex would be very helpful. Adding to that, converting from hex back to a basic image would be great with possible added features of desired image type and size.

**Comment by sharevb** [comment]
Hi @cmallory183 and @CorentinTh , implemented in #1263

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
